### PR TITLE
Update bazel to 0.11.1

### PR DESCRIPTION
**Pull request by sebastienvas**
Fixes https://github.com/istio/proxy/issues/1323

### CODE REVIEW

File: docker/istio_builders/CHANGELOG.md [hunk 1]
```diff
@@ -1,2 +1,3 @@
 * 0.5.1: update bazel to 0.11.0
 * 0.5.2: update golang to 1.10
+* 0.5.3: update bazel to 0.11.2
```
Comment (JimmyCYJ) [inline docker/istio_builders/CHANGELOG.md]: Is this going to be 0.11.1 instead of 0.11.2?

**Review by JimmyCYJ — COMMENTED** [review]
I have one question about istio_builders/CHANGELOG.md. LGTM overall.

**Comment by sebastienvas** [comment]
thanks for catching it.

**Review by JimmyCYJ — APPROVED** [review]

**Review by hklai — APPROVED** [review]

**Pull request merged**